modules/ROOT/pages/mongodb/mongodb-connector-6-0-design-center.adoc

```diff
@@ -0,0 +1,78 @@
+= MongoDB Design Center Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MongoDB Connector Version 6.0
+
+Design Center enables you to create apps visually. To use Design Center, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Design Center:
+
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
+
+For information on Design Center field values, see the xref:mongodb/mongodb-connector-reference.adoc[MongoDB Connector Reference].
+
+== Configure the Trigger
+
+. In Design Center, click *Create*.
+. Click *Create new application*.
+. Specify a value for *Project name*.
+. Exit from *Let's get started* by clicking *Go straight to canvas*.
+. Click the name of the trigger card.
+. To use the MongoDB connector as an input source trigger, choose the *Object Listener* input source.
```

Comment:
```suggestion
. To use Anypoint Connector for MongoDB as an input source trigger, choose the *Object Listener* input source.
```
First use of connector name in narrative text

Reply:
Kim, I haven't seen this type of comment before. Previously, we just used long name on first mention.